### PR TITLE
Add HTTP tags to ext/

### PR DESCRIPTION
**Pull request by bensigelman**
... also for the benefit of https://github.com/go-kit/kit/pull/247

### CODE REVIEW

File: ext/tags.go [hunk 1]
```diff
@@ -14,8 +14,33 @@ import opentracing "github.com/opentracing/opentracing-go"
...
+	//////////////////////////////////////////////////////////////////////
+	// SpanKind (client/server)
+	//////////////////////////////////////////////////////////////////////
+
+	// SpanKind hints at relationship between spans, e.g. client/server
+	SpanKind = spanKindTag("span.kind")
+
+	// SpanKindRPCClient marks a span representing the client-side of an RPC
+	// or other remote call
+	SpanKindRPCClient = SpanKindEnum("c")
```
Comment (yurishkuro) [inline ext/tags.go]: do you want to change them to client/server per Data Semantics, while at it?

**Comment by yurishkuro** [comment]
lgtm

File: ext/tags.go (hunk 1 above)
Comment (bensigelman) [inline ext/tags.go]: yes, good catch!

**Comment by bg451** [comment]
lgtm

**Pull request merged**